src/raw_decompiled/EDITOR2W.c

```diff
@@ -26587,7 +26587,7 @@ signed int __thiscall editManager::Main(editManager *this, int a2)
   {
     if ( gbDoRandomMap )
     {
-      editManage::GenerateRandomMap(this);
+      editManager::GenerateRandomMap(this);

```

Comment:
Thanks for fixing this typo.

I know you didn't write this, but....I'm surprised this compiles. It should be this->GenerateRandomMap();

Reply:
Isn't that an automatically generated file?

Reply:
Oh, I didn't realize this was EDITOR2W.c because I was reading bottom-up. So, this doesn't compile.